server/src/bots/telegram/botResponse/countriesResponse.ts

```diff
@@ -39,43 +40,63 @@ export const countriesResponse = (bot, message) => {
                     });
                 });
 
-            const portionSize: number = 40;
-
             bot.sendMessage(
                 getChatId(message),
                 `Total confirmed: ${worldTotalConfirmed}, recovered: ${worldTotalRecovered}, death: ${worldTotalDeaths} in ${countriesResult.length} countries.`
             );
 
+            const portionSize: number = 40;
+            const tableConfig = {
+                columns: {
+                  0: {
+                    width: 25
+                  }
+                },
+                columnDefault: {
+                    paddingLeft: 0,
+                    paddingRight: 0,
+                    width: 6
+                },
+                singleLine: true,
+                border: getBorderCharacters("honeywell")
+              };
+
             let portionMessage = [];
-            countriesResult
-                .forEach((countryResult: OverallCountrySituationResponse, idx: number) => {
-                    const {
-                        country,
-                        lastUpdateDate,
-                        totalConfirmed,
-                        totalRecovered,
-                        totalDeaths
-                    } = countryResult;
+            while(countriesResult.length > 0)
+            {
+                countriesResult
+                    .splice(0, portionSize)
```

Comment:
you're mutating array here - the side effect 

`const portionSize: number = 40;`
`let portionPage: number = 0;`
`let portion: Array<whatever> = countriesResult.slice(portionSize * portionPage++, portionSize)`
`while(!!portion?.length) { ...  portion: Array<whatever> = countriesResult.slice(portionSize * portionPage++, portionSize)`